src/encoded/static/components/item-pages/components/tables/ExperimentSetTables.js

```diff
@@ -76,23 +90,29 @@ ExperimentSetsTableTabView.getTabObject = function(props){
     };
 };
 
-export function ExperimentSetsTableTabViewTitle({ totalCount, searchHref }) {
-    const linkText =
-        searchHref && typeof searchHref === 'string' && searchHref.indexOf('/browse/') > -1 ?
-            'Open In Browse View' : 'Open In Search View';
+/**
+ * @todo Eventually maybe add UI controls for selecting columns and other things into here.
+ */
+export function ExperimentSetsTableTabViewTitle(props) {
+    const {
+        totalCount,
+        href: currentSearchHref,
+        externalSearchLinkVisible = false
```

Comment:
Same as setting defaultProps.externalSearchLinkVisible to false pretty much. Setting default to false here since seems is most common case and we currently only show it on ExperimentTypeView table.

Reply:
@alexkb0009 I think there is a misunderstanding. The default value of `externalSearchLinkVisible` was `true` (not `false`). We set it as false in some pages like FileView. So, we are not displaying the button where we actually should.